ev3dev2/motor.py

```diff
@@ -1920,10 +2003,174 @@ def get_speed_steering(self, steering, speed):
             right_speed *= speed_factor
         else:
             left_speed *= speed_factor
-        
+
         return (left_speed, right_speed)
 
 
+class MoveDifferential(MoveTank):
```

Comment:
The other classes inherit from MoveTank, but honestly I think I would prefer composition over inheritence. This is a peer to MoveTank so while it seems reasonable to use that other class's behavior I think any common stuff should be either re-exposed manually or moved up one level in the tree. The inheritance makes documentation pretty tough.

Reply:
What do you say we tackle how to organize these as part of the issue regarding a potential drive.py?

Reply:
This is basically MoveTank but with known wheel circumference and wheel distance which makes more things possible. It is adding to MoveTank but can still use all of the functionality provided by MoveTank. To me this is very much a child class not a peer class.

Reply:
OK, I see what you're saying; the `MoveTank` methods aren't being replaced here so you can still use them. I think it makes sense now. I was thinking this was more like the others but that's a different case.